iree-prof-tools/iree-prof-output.cc

```diff
@@ -32,7 +32,14 @@ ABSL_FLAG(bool, output_ops_stdout, true,
           "Whether to print Tracy result of ML operation to stdout.");
 ABSL_FLAG(std::string, zone_regex,
           "iree_hal_buffer_map_(zero|fill|read|write|copy)|_dispatch_[0-9]+",
-          "ECMAScript regex of tracy zones to output to stdout.");
+          "ECMAScript regex of tracy zones to output to stdout. If empty, no "
+          "zones will be matched with regex. Note that zones can still be "
+          "matched with --zone_substrs flag.");
+ABSL_FLAG(std::vector<std::string>, zone_substrs, {},
```

Comment:
Using substr is faster, so it would be better to use it by default instead of regex.

Reply:
done